applications/FluidDynamicsApplication/python_scripts/navier_stokes_two_fluids_solver.py

```diff
@@ -139,13 +159,21 @@ def Initialize(self):
             computing_model_part,
             computing_model_part.ProcessInfo[KratosMultiphysics.DOMAIN_SIZE])
 
-        # Elemental neighbours calculation
+        # Finding nodal and elemental neighbors
         data_communicator = computing_model_part.GetCommunicator().GetDataCommunicator()
-        neighbour_search = KratosMultiphysics.FindGlobalNodalElementalNeighboursProcess(
+        neighbour_search = KratosMultiphysics.FindGlobalNodalNeighboursProcess(
             data_communicator,
             computing_model_part)
         neighbour_search.Execute()
 
+        dimensions = computing_model_part.ProcessInfo.GetValue(KratosMultiphysics.DOMAIN_SIZE)
+        avg_num_elements = dimensions + 1
```

Comment:
Why changing it to dim + 1? I'd leave the default value.

Reply:
"avg_num_elements" is needed based on https://github.com/KratosMultiphysics/Kratos/blob/2714fc72985d61fa52b65533eabb6cb8b091b823/kratos/python/add_processes_to_python.cpp#L188
However, the definition allows let it be the default value:
https://github.com/KratosMultiphysics/Kratos/blob/2714fc72985d61fa52b65533eabb6cb8b091b823/kratos/processes/find_elements_neighbours_process.h#L82
Am I right?

Reply:
Yes, it is required. But my point is why you changed it to `dim+1` rather than keeping 10.

Reply:
Done.